tsfile/src/main/java/org/apache/iotdb/tsfile/file/header/ChunkHeader.java

```diff
@@ -153,6 +153,8 @@ public static ChunkHeader deserializeFrom(InputStream inputStream, byte chunkTyp
       throws IOException {
     // read measurementID
     String measurementID = ReadWriteIOUtils.readVarIntString(inputStream);
+    measurementID =
+        measurementID == null ? "" : measurementID; // measurementID in TimeChunk header is null
```

Comment:
1. Comments should be placed on the top line of code
2. Why is this parameter set to "" when measurementId is empty?